proctor-codegen/src/main/resources/com/indeed/proctor/consumer/ant/groups.ftl

```diff
@@ -149,7 +149,7 @@ public class ${mainClassName} extends AbstractGroups {
     <#if (testDef.payloadJavaClass)??>
     <#if (testDef.isMap)??>
     public @Nullable ${mainClassName}Payload.${testDef.name?cap_first} get${testDef.javaClassName}Payload() {
-        final Payload payload = getPayload(${testEnumName}.${testDef.enumName}.getName()<#if testDef.buckets?has_content>, ${testDef.javaClassName}.getFallback()</#if>);
+        final Payload payload = getPayload(${testEnumName}.${testDef.enumName}.getName(), ${testEnumName}.${testDef.enumName}.getFallbackValue());
```

Comment:
What is the difference between `.enumName` and `.javaClassName`?

Reply:
For example, for the test `"payloadonly_doubletype"`, `.enumName` is `Test.BUBBLE` and `.javaClassName` is `Bubble`. For payload-only tests, no classes implementing `Bucket<Test>` are generated.
```    
    public enum Test implements com.indeed.proctor.consumer.Test {
        BUBBLE("bubble", 0),
        ...
        ; // fix compilation if no tests

        private final String name;
        private final int fallbackValue;

        private Test(final String name, final int fallbackValue) {
            this.name = name;
            this.fallbackValue = fallbackValue;
        }

        @Override
        public String getName() {
            return name;
        }
        @Override
        public int getFallbackValue() {
            return fallbackValue;
        }
    }

    public enum Bubble implements Bucket<Test> {
        CONTROL(0, "control"),
        TEST(1, "test"),
        OOP(2, "oop"),
        POOP(3, "poop");

        private final int value;
        private final String name;
        private final String fullName;
        private Bubble(final int value, final String name) {
            this.value = value;
            this.name = name;
            this.fullName = getTest().getName() + "-" + name;
        }

        @Override
        public Test getTest() {
            return Test.BUBBLE;
        }

        @Override
        public int getValue() {
            return value;
        }

        @Override
        public String getName() {
            return name;
        }

        @Override
        public String getFullName() {
            return fullName;
        }

        public static Bubble getFallback() {
            return Bubble.CONTROL;
        }
    }
```

Reply:
I see. That makes sense.